tests/foreman/endtoend/test_cli_endtoend.py

```diff
@@ -54,59 +54,52 @@
 from robottelo.constants.repos import FAKE_0_PUPPET_REPO
 from robottelo.decorators import setting_is_set
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import CLITestCase
 
 
-class EndToEndTestCase(CLITestCase, ClientProvisioningMixin):
+class TestEndToEnd(ClientProvisioningMixin):
```

Comment:
Is there a need to keep these tests under a class?

Reply:
Yes, same here as well, with this PR trying to keep the same structure for all end-to-end tests under `/tests/foreman/endtoend`, like keeping a class, and tests inside a class, as similar to https://github.com/SatelliteQE/robottelo/pull/8430 , also it needs a rebase over the PR, whichever gets merged first for changes to `.endtoend/utils.py` :)

Reply:
So this end to end test file is different than the one @mshriver refactored. In his, classes were used to organize different types of tests. However, in this there is just the one.

Reply:
@JacobCallahan Ack, removed class and made all these tests module level, Request to review again

Updated tests name as well to use `cli_` and keeping the old `test_ids` for tests, which were updated earlier by pre-commit.